js/ui/toolbar/internal/ui.toolbar.menu.js

```diff
@@ -218,6 +217,9 @@ export default class DropDownMenu extends Widget {
             showTitle: false,
             fullScreen: false,
             _fixWrapperPosition: true,
+            wrapperAttr: {
+                role: 'menu',
```

Comment:
WAVE reports about wrapper's child without role `menuitem` and about item's parent without role `menu`. Please, check it